get-started/next-steps-resources.md

```diff
@@ -0,0 +1,178 @@
+# Next steps & resources
```

Comment:
this is not clearly describing what this page covers. why did we choose to split this onto its own page?